dbt/models/eia860/_core_eia860__cooling_equipment/schema.yml

```diff
@@ -4,6 +4,15 @@ sources:
     tables:
       - name: _core_eia860__cooling_equipment
         data_tests:
+          - expect_columns_not_all_null:
+              exclude_columns:
+                - tower_type_3
+                - tower_type_4
+                - cooling_type_4
```

Comment:
Appear to be placeholders for a larger number of these kinds of things than anyone is actually using right now.  But would be nice if we could automatically detect if/when someone does actually report a value in one of them so we can make this more stringent / migrate it to a conditional column rather than an exclusion.

I guess we could also just use the conditional columns to exclude all years up to the present, and each year that would result in a new failure in the new year, unless someone has started using the column.  Maybe that would be more elegant / simpler.

Reply:
@krivard ☝🏼 this seemed like it might be a good way to handle the "nobody has used this column **yet**" cases such that we wouldn't forever exclude a column that hasn't been used.  But also there's a good chance that these 3rd and 4th tier categories will just end up being the kind of column that are super sparsely used and often have years with no values, in which case allowing them to be fully NULL seems fine.